packages/react/src/hooks/useIsClient.spec.tsx

```diff
@@ -51,7 +55,7 @@ describe('useIsClient', () => {
     expect(mockUseIsClient).toBeCalledTimes(2)
   })
   it('improve legacy useIsClientOnly', () => {
-    const { useIsomorphicLayoutEffect } = importUseIsomorphicLayoutEffect()
+    const { useClientLayoutEffect } = importUseClientLayoutEffect()
```

Comment:
Our legacy code was useIsomorphicLayoutEffect right. so if we should compare correctly. we need to keep this